test/spec/downloadTests.ts

```diff
@@ -26,16 +31,54 @@ if (fs.existsSync(outputDir)) {
 
 fs.mkdirSync(outputDir, {recursive: true});
 
-for (const test of SPEC_TEST_TO_DOWNLOAD) {
-  const url = `${specTestsRepoUrl}/releases/download/${specVersion}/${test}.tar.gz`;
-  console.log(`Downloading ${url} to ${outputDir}`);
-  shell(`wget -c ${url} -O - | tar -xz --directory ${outputDir}`);
-  console.log(`Downloaded ${url} to ${outputDir}`);
-}
+const urls = SPEC_TEST_TO_DOWNLOAD.map((test) => `${specTestsRepoUrl}/releases/download/${specVersion}/${test}.tar.gz`);
 
-// If successful
-fs.writeFileSync(versionFile, specVersion);
+downloadAndExtract(urls, outputDir)
+  .then(() => {
+    console.log("Downloads and extractions complete.");
```

Comment:
Log success and error for each file individually downloads take very long time

Reply:
I think that might clutter the logs and make it difficult to detect an error, I would prefer to log an error should that occur during the process. 

The Total size is about 17.4MB when unzipped, so I don't think it should take particularly long with average bandwidths

Reply:
Oh right I was thinking of mainnet spec tests